web-ui/tslint.json

```diff
@@ -15,7 +15,6 @@
     "forin": true,
     "import-blacklist": [
       true,
-      "rxjs",
```

Comment:
Why is this deleted?

Reply:
In the newest version of RxJS, the way to import objects [has changed](https://angular.io/guide/rx-library) and importing directly `from 'rxjs'` is not considered as a bad practice anymore.